analyzer/dpd.sig

```diff
@@ -0,0 +1,23 @@
+signature dpd_ldap_client_udp {
```

Comment:
We should really add a test for DPD. Like you mentioned offline, we could rewrite the PCAP we already have in tree to use a different port and check whether we get similar results.

Reply:
Added